modules/gltf/src/gltf/gltf-parser.js

```diff
@@ -26,13 +27,13 @@ export default class GLTFParser {
       await this._loadLinkedAssets(options);
     }
 
-    if (options.decompress) {
-      this._decompressMeshes(options);
-    }
-
     if (options.postProcess) {
       const postProcessor = new GLTFPostProcessor();
-      postProcessor.postProcess(gltf, options);
+      postProcessor.postProcess(this.gltf, this.glbParser, options);
+
+      if (options.decompress) {
```

Comment:
The postProcessing needs to be an optional step independent of decompression. We should be able to move postProcessing out of this module without breaking anything. If there is something the postProcessing is doing that is needed for decompression we need to move that back into the Parser.